### PR TITLE
MEN-5461 Notes on removing the stable APT repository.

### PR DESCRIPTION
**Pull request by merlin-northern**
ChangeLog: none
Signed-off-by: Peter Grzybowski <peter@northern.tech>

### CODE REVIEW

File: 301.Troubleshoot/03.Mender-Client/docs.md [hunk 1]
```diff
@@ -4,6 +4,23 @@ taxonomy:
     category: docs
 ---
 
+## Removed previous stable APT repositories
+
+We [removed](../../09.Downloads/docs.md#Set-up-the-APT-repository) the previoussly deprecated stable APT repository:
```
Comment (oleorhagen) [inline 301.Troubleshoot/03.Mender-Client/docs.md:9]: `s/previoussly/previously/`

File: 09.Downloads/docs.md [hunk 2]
```diff
@@ -166,6 +166,15 @@ repository. Afterwards, you can install and update the Mender client using the
 
 ##### Set up the APT repository
 
+<!--AUTOVERSION: "Mender %"/ignore -->
+!!! As of Mender 3.2.1 we deprecated previous stable repository and stopped updating it.
```
Comment (oleorhagen) [inline 09.Downloads/docs.md:170]: S:
`deprecated _the_ previous`

Comment (oleorhagen) [inline 09.Downloads/docs.md:170]: S:
`deprecated _the_ previous`

File: 09.Downloads/docs.md [hunk 3]
```diff
@@ -166,6 +166,15 @@ repository. Afterwards, you can install and update the Mender client using the
...
+<!--AUTOVERSION: "Mender %"/ignore -->
+!!! As of Mender 3.3 we removed it.
+!!! Please use, for instance, `https://downloads.mender.io/repos/debian debian/buster/stable main` and _not_
```
Comment (oleorhagen) [inline 09.Downloads/docs.md:173]: S:

Remove, `for instance`

Comment (oleorhagen) [inline 09.Downloads/docs.md:173]: S:

Remove, `for instance`

File: 09.Downloads/docs.md [hunk 4]
```diff
@@ -166,6 +166,15 @@ repository. Afterwards, you can install and update the Mender client using the
 
 ##### Set up the APT repository
 
+<!--AUTOVERSION: "Mender %"/ignore -->
+!!! As of Mender 3.2.1 we deprecated previous stable repository and stopped updating it.
+<!--AUTOVERSION: "Mender %"/ignore -->
+!!! As of Mender 3.3 we removed it.
+!!! Please use, for instance, `https://downloads.mender.io/repos/debian debian/buster/stable main` and _not_ 
+!!! "https://downloads.mender.io/repos/debian stable main".
+!!! Please make sure you have the currently supported version of your choice in place and please
```
Comment (oleorhagen) [inline 09.Downloads/docs.md:175]: Q:


Maybe we should have a description of how to actually upgrade it? Like with examples and such?

Comment (oleorhagen) [inline 09.Downloads/docs.md:175]: Q:


Maybe we should have a description of how to actually upgrade it? Like with examples and such?

Comment (merlin-northern) [reply 09.Downloads/docs.md:175]: good idea. added!

File: 301.Troubleshoot/03.Mender-Client/docs.md (hunk 1 above)
Comment (merlin-northern) [reply 301.Troubleshoot/03.Mender-Client/docs.md]: thanks.. ssorry about that.

File: 09.Downloads/docs.md (hunk 3 above)
Comment (merlin-northern) [reply 09.Downloads/docs.md]: removed.

File: 09.Downloads/docs.md (hunk 2 above)
Comment (merlin-northern) [reply 09.Downloads/docs.md]: fixed. fixed.

Comment (merlin-northern) [reply 09.Downloads/docs.md]: fixed.

File: 09.Downloads/docs.md (hunk 4 above)
Comment (merlin-northern) [reply 09.Downloads/docs.md:175]: is it only me or every comment is doubled? or you wanted to make a point? ;-)

**Review by oleorhagen — APPROVED** [review]

File: 301.Troubleshoot/03.Mender-Client/docs.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (merlin-northern) [reply 301.Troubleshoot/03.Mender-Client/docs.md]: stale comment, discussed privately, had been fixed.

**Pull request merged**